ds3-autogen-java/src/test/java/com/spectralogic/ds3autogen/java/generators/responsemodels/BaseResponseGenerator_Test.java

```diff
@@ -94,12 +60,11 @@ public void getAllImports_FullList_Test() {
                 createEmptyDs3Request(),
                 responseCodes,
                 "com.spectralogic.ds3client.commands.spectrads3");
-        assertThat(result.size(), is(5));
+        assertThat(result.size(), is(4));
```

Comment:
I moved the associated logic from the Response parser into a util function. As I did that, I made modifications so that the generated code is as expected for the response parsers.  Eventually this function may be removed completely from the response generation when the responses are converted into simple POJOs.  Therefore, I simply updated the test to pass for now knowing that a separate refactor for the request handlers is going to take place as part of the overall refactor.